general/1_terminal/Tima-Muravyov/README.md

```diff
@@ -1,10 +1,6 @@
 # homework 1
 
 
-![](images/task1.png)
```

Comment:
Do you need this change?

Reply:
it will be better without image, saeryan said that and I agree cause repo will take less space